forest/src/cli/mod.rs

```diff
@@ -68,6 +68,12 @@ pub struct DaemonOpts {
     pub rpc: Option<bool>,
     #[structopt(short, long, help = "The port used for communication")]
     pub port: Option<String>,
+    #[structopt(
+        short,
+        long,
+        help = "Port used for Prometheus metric collection server"
+    )]
+    pub prometheus_port: Option<u16>,
```

Comment:
I'd call this `metrics_port`. Would be easier to people to understand what it is if they dont know prometheus. Or if we move away from prometheus later, the flag doesnt need to change

Reply:
(Thats a nit btw, so dont worry about it)